@here/harp-webtile-datasource/lib/WebTileDataSource.ts

```diff
@@ -413,13 +413,13 @@ export class WebTileDataSource extends DataSource {
  * Definitions of variable values to be used with `WebTileDataSource`
  */
 export namespace WebTileDataSource {
```

Comment:
Not to to readers why we _still_ leave `namespace` and `class` merging.

We still didn't ban them and they are described in [Typescript Handbook](https://www.typescriptlang.org/docs/handbook/declaration-merging.html) as legitimate language feature.

Let's remove them later and not only fix the bug.